mysql-test/suite/sys_vars/r/wsrep_provider_options_basic.result

```diff
@@ -42,8 +45,9 @@ SET @@global.wsrep_provider_options=NULL;
 Got one of the listed errors
 SELECT @@global.wsrep_provider_options;
 @@global.wsrep_provider_options
-NULL
+
 
 # restore the initial value
 SET @@global.wsrep_provider_options = @wsrep_provider_options_global_saved;
+ERROR HY000: WSREP (galera) not started
```

Comment:
why we will get this error , when we restore the original option ? , Sorry I did not understand the point of this test case

Reply:
The wsrep provider (i.e. Galera library) is not loaded while this test is executed. 
This patch changes the behavior of this variable, so that it does not allow setting provider options, when no provider is loaded.

Avoiding weird situations, such as the following:
```
SET @@global.wsrep_provider_options=NULL;
Got one of the listed errors
SELECT @@global.wsrep_provider_options;
@@global.wsrep_provider_options
NULL
```
Where previously you could set provider options, get an error back, and the value changed!

Reply:
Also, notice that the variable never changed its value during the test.

Reply:
Fixup in commit 03b11c2

Reply:
Ohh okay , thanks for patch